tsconfig.json

```diff
@@ -13,11 +13,10 @@
     "strictNullChecks": true
   },
   "include": [
-    "./src/**/*",
-    "./e2e/**/*"
+    "./src/**/*"
```

Comment:
this is a bit strange that we build test to dist dir and then run them. I've never seen such projects before. is it difficult to change it so we can run tests from the test dir?